src/cadence/mainnet/getNFTs.cdc

```diff
@@ -1486,4 +1490,66 @@ pub fun getMiamiNFT(owner: PublicAccount, id: UInt64): NFTData? {
             "season": miamiData.season
         },
     )
+}
+
+// https://flow-view-source.com/mainnet/account/0xe4cf4bdc1751c65d/contract/AllDay
+pub fun getAllDay(owner: PublicAccount, id: UInt64): NFTData? {
+    let contract = NFTContract(
+        name: "AllDay",
+        address: 0xe4cf4bdc1751c65d,
+        storage_path: "AllDay.CollectionStoragePath",
+        public_path: "AllDay.CollectionPublicPath",
+        public_collection_name: "AllDay.MomentNFTCollectionPublic",
+        external_domain: "https://nflallday.com/"
+    )
+
+    let col = owner.getCapability(AllDay.CollectionPublicPath)
+        .borrow<&{AllDay.MomentNFTCollectionPublic}>()
+    if col == nil { return nil }
+
+    let nft = col!.borrowMomentNFT(id: id)
+    if nft == nil { return nil }
+
+    return NFTData(
+        contract: contract,
+        id: nft!.id,
+        uuid: nil,
+        title: "Moment".concat(nft!.id.toString()).concat("-Edition").concat(nft!.editionID.toString()).concat("-SerialNumber").concat(nft!.serialNumber.toString()),
```

Comment:
Since most of our Moment metadata will be fetched from a separate API I left most of these blank, but filled in the title with a concatenated string of the MomentID, EditionID, and SerialNumber